src/main/webapp/dev/js/main/adminSearch.es6

```diff
@@ -144,4 +145,56 @@ $(document).ready(() => {
     $('#btn-collapse-all-students').on('click', () => {
         adminSearchCollapseAllStudents();
     });
+
+    $('.openEmailApplicationDefaultValues').submit((e) => {
+        e.preventDefault();
+        const $entry = $(e.target);
+        const studentEmail = $entry.parentsUntil('.fslink_student').find('input[name=studentEmail]').val();
+        const subjectType = $entry.find('input[name=subjectType]').val();
+        const courseName = $entry.find('input[name=courseName]').val();
+        const courseId = $entry.find('input[name=courseId]').val();
+        const studentName = $entry.find('input[name=studentName]').val();
+        const sessionStatus = $entry.find('input[name=sessionStatus]').val();
+        const sessionName = $entry.find('#feedback-session-name').text();
+        const sessionNameWithoutDate = sessionName.substring(0, sessionName.lastIndexOf(' ['));
+
+        const relatedLink = encodeURIComponent($entry.find('input[name=relatedLink]').val());
+        const googleSignup = encodeURIComponent('https://accounts.google.com/NewAccount');
+        const supportEmail = `${Const.TEAMMATES_EMAIL_ADDRESS}`;
+
+        const bodyHeader = `Hello ${studentName},`;
+        const bodyFooter =
+                '%0D%0A%0D%0AIf you encounter any problems when using the system,'
+                + ` you can email TEAMMATES support team at ${supportEmail}.`
+                + '%0D%0A%0D%0ARegards,%0D%0ATEAMMATES Team.';
+
+        const uniqueLinkMessage = 'The above link is unique to you. Please do not share it with your classmates.';
+
+        let subject = '';
+        let bodyContent = '';
+
+        if (subjectType === 'Invitation to join course') {
```

Comment:
Possible future work here.

This entire construction of `subject` and `bodyContent` are essentially repetitions of the email templates defined [here](https://github.com/TEAMMATES/teammates/tree/master/src/main/resources). There should be a way to get the information from the back-end so as to make this repetition not needed.

Of course, it may help to even revamp the entire email templating system.